### PR TITLE
E2E tests for DeltaToken and Untyped DeltaFeedSerialization

### PR DESCRIPTION
**Pull request by hotchandanisagar**
E2E tests for DeltaToken and Untyped DeltaFeedSerialization

### CODE REVIEW

File: OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        [InlineData("application/json;odata.metadata=minimal")]
+        [InlineData("application/json;odata.metadata=full")]
+        public void UntypedDeltaWorksInAllFormats(string acceptHeader)
+        {
+            string url = "/untyped/UntypedDeltaCustomers?$deltatoken=abc";
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, BaseAddress + url);
+            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse(acceptHeader));
+            HttpResponseMessage response = Client.SendAsync(request).Result;
+            Assert.True(response.IsSuccessStatusCode);
+            Assert.NotNull(response.Content);
```
Comment (xuzhg) [inline OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs]: Can you verify the content detail?

File: OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            customers.EntityType.Property(c => c.Name).IsRequired();
+            var orders = builder.EntitySet<UntypedOrder>("UntypedDeltaOrders");
+            return builder.GetEdmModel();
+        }
+
+        [Theory]
+        [InlineData("application/json")]
+        [InlineData("application/json;odata.metadata=minimal")]
+        [InlineData("application/json;odata.metadata=full")]
+        public void UntypedDeltaWorksInAllFormats(string acceptHeader)
```
Comment (xuzhg) [inline OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs]: Can you add negative scenario?

Comment (hotchandanisagar) [inline OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs]: As discussed with you, handling of Delta Token is the responsibility of the Server, so could not have a negative scenario here

File: OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs (hunk 1 above)
Comment (hotchandanisagar) [inline OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs]: I have updated the Test case with the verification of content detail

**Pull request merged**